index.php

```diff
@@ -2,28 +2,32 @@
     $page['title'] = 'Purchase elementary OS';
 
     $page['script-plugins'] = array(
+        'https://cdn.jsdelivr.net/bluebird/3.4.1/bluebird.min.js',
         'https://cdn.jsdelivr.net/g/jquery.leanmodal2@2.5',
-        'scripts/slider.js'
+        'scripts/showcase.js',
+        'scripts/terminal.js'
     );
 
     $page['scripts'] = array(
         'https://checkout.stripe.com/checkout.js' => array(
             'data-alipay' => 'auto',
-            'data-locale' => 'auto'
+            'data-locale' => 'auto',
+            'async' => false
```

Comment:
I would guess without it, it would be possible for the download script to load before it and error on payment, but I have yet to test it with `async`.